src/test/java/net/snowflake/client/jdbc/telemetryOOB/TelemetryServiceIT.java

```diff
@@ -231,4 +235,69 @@ public void stressTestCreateUrgentEvent()
     }
     sw.stop();
   }
+
+  private void generateDummyException(int value) throws SnowflakeSQLLoggedException {
+    if (value > 0)
+    {
+      String queryID = "01234567-1234-1234-1234-00001abcdefg";
+      String reason = "This is a test exception.";
+      String sqlState = SqlState.NO_DATA;
+      int vendorCode = 0;
+      throw new SnowflakeSQLLoggedException(queryID, reason, sqlState, vendorCode);
+    }
+  }
+  
+  /**
+   * Manual test case for checking telemetry message for SnowflakeSQLExceptions. Connect to dev or prod account and
+   * query client_telemetry table in s3testaccount for results.
+   * @throws SQLException
+   */
+  @Ignore
+  @Test
+  public void testSnowflakeSQLLoggedExceptionTelemetry() throws SQLException
+  {
+    // make a connection to initialize telemetry instance
+    Connection con = getConnection();
```

Comment:
Can you also check what happens if no connection is created and then throw an exception?

Reply:
Not making a connection throws a SQLException with info about why the connection was not made, so I think it's okay to just put the SQLException in the method signature and let it be thrown if it happens.

Reply:
Can you add a test case for that? If that happens, the log is not going to send to our side, how to we notice it and then fix it?